src/code-templates/_shared/MethodBody/PathParameter.ts

```diff
@@ -18,25 +18,23 @@ const generateUrlVariableStatement = (
   urlTemplate: Utils.Params$TemplateExpression,
   methodType: MethodType,
 ): ts.VariableStatement => {
-  const left: Record<MethodType, ts.Expression> = {
-    class: factory.PropertyAccessExpression.create({
-      name: "baseUrl",
-      expression: "this",
-    }),
-    function: factory.Identifier.create({
-      name: "_baseUrl",
+  const expression = {
+    class: factory.BinaryExpression.create({
+      left: factory.PropertyAccessExpression.create({
+        name: "baseUrl",
+        expression: "this",
+      }),
+      operator: "+",
+      right: Utils.generateTemplateExpression(factory, urlTemplate),
     }),
+    function: Utils.generateTemplateExpression(factory, urlTemplate),
```

Comment:
I can't understand the reason but this code isn't worked.

The `class` part works fine but `function` part will be wrong template string.